src/autocomplete.js

```diff
@@ -341,15 +346,26 @@ class Autocomplete {
 
             if (finished) {
                 // No results
-                if (!filtered.length)
+                if (!filtered.length && (!this.emptyMessage || this.autoShown))
                     return this.detach();
 
+                if (!filtered.length && this.emptyMessage && !this.autoShown) {
+                    var completionsForEmpty = [{
+                        caption: this.emptyMessage(prefix),
+                        value: ""
+                    }];
+                    this.completions = new FilteredList(completionsForEmpty);
+                    this.completions.exactMatch = true;
```

Comment:
Why is exactMatch needed here, is it something that was needed for the old pull request?